contracts/mocks/ERC1363PayableMock.sol

```diff
@@ -15,12 +15,12 @@ contract ERC1363PayableMock is ERC1363Payable {
     constructor(IERC1363 acceptedToken) ERC1363Payable(acceptedToken) {} // solhint-disable-line no-empty-blocks
 
     // solhint-disable-next-line no-unused-vars
-    function _transferReceived(address operator, address from, uint256 value, bytes memory data) internal override {
+    function _transferReceived(address /*operator*/, address /*from*/, uint256 /*value*/, bytes memory /*data*/) internal override {
```

Comment:
Please remove instead of commenting and also remove the above solhint disable

```suggestion
    function _transferReceived(address, address, uint256, bytes memory) internal override {
```

Reply:
This is not the recommend change, and our internal coding standards require more documentation, not less.  We plan on releasing our own token under this project, and we'll have to add in more code comments before full approval.

Reply:
ERC1363PayableMock.sol is a mock contract used only for tests. If you require more comments feel free to fork and edit as per your requirements.

Reply:
Two steps ahead of you, we where pushing back changes to help out the community.